content/python/concepts/random-module/terms/random/random.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: '.random()'
+Description: 'Returns a pseudo-random floating-point number between 0 and 1.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Random'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+The `random.random()` method accepts no parameters and returns a pseudo-random floating-point number between 0.0 and 1.0.
+
+## Syntax
+
```

Comment:
```suggestion

After importing the `random` module, the `random.random()` method can be accessed:

```

Reply:
This is totally not necessary but was wondering everyone else thought about at least mentioning this in the context of applying the syntax?

Reply:
@Dusch4593, I don't know. Isn't it implied by being an entry under "random-module?"